### ISSUE TITLE
Logic for sample numbering is incorrect

### ISSUE THREAD

**Issue by JohannesGaessler** [hlky/stable-diffusion-webui#112]
This is the sample numbering that I got when using this branch https://github.com/hlky/stable-diffusion-webui/pull/111 to generate samples:
```
0755    - johannesg johannesg 27 Aug 08:24 -N  Flowers,_red,_blue/
0644 518k johannesg johannesg 27 Aug 07:59 -N ├──  -0001-0.png
0644 518k johannesg johannesg 27 Aug 08:22 -N ├──  00000-0.png
0644  187 johannesg johannesg 27 Aug 08:22 -N ├──  00000-0.yaml
0644 471k johannesg johannesg 27 Aug 08:22 -N ├──  00001-1.png
0644  187 johannesg johannesg 27 Aug 08:22 -N ├──  00001-1.yaml
0644 476k johannesg johannesg 27 Aug 08:23 -N ├──  00002-2.png
0644  187 johannesg johannesg 27 Aug 08:23 -N ├──  00002-2.yaml
0644 462k johannesg johannesg 27 Aug 08:24 -N ├──  00003-3.png
0644  187 johannesg johannesg 27 Aug 08:24 -N ├──  00003-3.yaml
0644 496k johannesg johannesg 27 Aug 08:24 -N ├──  00004-4.png
0644  187 johannesg johannesg 27 Aug 08:24 -N └──  00004-4.yaml
```
It seems the indices need to be increased by 1.
I didn't touch indices in my PR so presumably the current master branch has the same problem.

**Comment by hlky** [comment]
Ok so you want it to start at 1? or?

**Comment by JohannesGaessler** [comment]
I think there has been a misunderstanding.
What I posted is an excerpt from my terminal when listing the files in a directory.
I was running a batch generation at the time and did not want to interrupt it to do software development, so I just opened a low-effort, quick issue instead; perhaps I should have been more explicit about what the problem is.

Notice that the file name in the second line is `-0001-0.png`.
In other words, the index begins at -1 instead of 0.
